addons/web/data/report_layout.xml

```diff
@@ -1,36 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
-        <record id="report_layout_standard" model="report.layout">
-            <field name="name">Standard</field>
-            <field name="view_id" ref="web.external_layout_standard"/>
-            <field name="image">/web/static/img/preview_standard.png</field>
-            <field name="pdf">/web/static/pdf/preview_standard.pdf</field>
-        </record>
-        <record id="report_layout_background" model="report.layout">
+
+         <record id="report_layout_background" model="report.layout">
             <field name="name">Background</field>
             <field name="view_id" ref="web.external_layout_background"/>
             <field name="image">/web/static/img/preview_background.png</field>
             <field name="pdf">/web/static/pdf/preview_background.pdf</field>
         </record>
+
```

Comment:
are you sure about the changes ?
those seem to just move around definitions of records

Reply:
This order defines the order in the view. Changes were requested so the default one would be first.

Reply:
@SimonGenin 
hmmmmm, this is fairly weak. I'd would have preferred a new field "sequence" on report.layout

Reply:
I can look into that